lib/github/filter.go

```diff
@@ -0,0 +1,40 @@
+package github
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/antonmedv/expr"
+)
+
+// RepositoryFilterEnv filter environment for repositories
+type RepositoryFilterEnv struct {
+	Repositories []Repository
+}
+
+// Contains reports wether substring is in s.
+func (RepositoryFilterEnv) Contains(s, substring string) bool {
```

Comment:
This allows for easy extension of more advanced predefined filters to be used in expressions.